### ISSUE TITLE
Maybe incorrect statistics on the report chart

### ISSUE THREAD

**Issue by yunwu-cn** [serenity-bdd/serenity-maven-plugin#85]
Hi I was using cucumber-serenity for some tests, it runs great with awesome report view.
However, when I select some tags, the stats seem incorrect on the overview piechart.  

Here's the screenshot of my report, when I select tag: Dev.
There should be only 2 tests. But the Test Outcomes showed 4, and the Overview piechart center has 200%, which will confuse people. 
![image](https://user-images.githubusercontent.com/92051495/145164990-5080c33d-c192-4def-88c4-c0b7ccbb5ab7.png)

Here's how the feature file looks like:

```
Feature: some feature name

  Scenario Outline: Positive Test 
    Given ***
    When ***
    Then ***

    @dev
    Examples:
     | username  | password  |
     | user1     | password1 |

    @qa
    Examples:
     | username  | password  |
     | user2     | password2 |

  Scenario Outline: Negative Test 
    Given ***
    When ***
    Then ***

    @dev
    Examples:
     | username  | password  |
     | user3     | password3 |

    @qa
    Examples:
     | username  | password  |
     | user4     | password4 |
```

When run the test, I do `mvn verify -Dcucumber.filter.tags="@dev or @qa"`

**Comment by wakaleo** [comment]
Yeah, the percentage looks wrong. The number of test outcomes will include rows of test data, which may be why the test outcomes is reporting 4. Can you provide a sample project to reproduce the issue?

**Comment by yunwu-cn** [comment]
Thank you for such a quick reply. I will provide a sample project tomorrow here. 

> Yeah, the percentage looks wrong. The number of test outcomes will include rows of test data, which may be why the test outcomes is reporting 4. Can you provide a sample project to reproduce the issue?

**Comment by wakaleo** [comment]
If you want to take a look at the code yourself, the graph uses jscharts and is in the [result-chart.ftl](https://github.com/serenity-bdd/serenity-core/blob/master/serenity-report-resources/src/main/resources/freemarker/components/result-chart.ftl) file - it could be a simple fix.

**Comment by yunwu-cn** [comment]
Will take a look tomorrow if I get time. Thank you.

> If you want to take a look at the code yourself, the graph uses jscharts and is in the [result-chart.ftl](https://github.com/serenity-bdd/serenity-core/blob/master/serenity-report-resources/src/main/resources/freemarker/components/result-chart.ftl) file - it could be a simple fix.

**Comment by yunwu-cn** [comment]
@wakaleo  I haven't got time to dive too deep. But it looks like the ResultCounts that passed into the page, the counts are only filtered by Result, not tag.
Here's my sample project that can be used to reproduce the issue:
https://github.com/yunwu-cn/sample-serenity-report-issue

Just run `mvn verify` to generate the HTML report. And go to the bottom of the report, click one of the tags, you will see the incorrect stats near the top of the page.

**Comment by wakaleo** [comment]
Could you try with 3.2.0?

**Comment by yunwu-cn** [comment]
tried with 3.2.0, still see the same issue. ps, above [sample project](https://github.com/yunwu-cn/sample-serenity-report-issue) is updated to use 3.2.0.

**Comment by wakaleo** [comment]
The reports with 3.2.0 seem to be as expected - what are you expecting to be displayed?
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/55986/154535061-b85c01f1-67cb-4129-8014-891f6fd0373f.png">

<img width="1835" alt="image" src="https://user-images.githubusercontent.com/55986/154535137-d30b0e45-167b-43fb-abac-d1b4ab360980.png">

<img width="1892" alt="image" src="https://user-images.githubusercontent.com/55986/154535203-7b021eb6-f97f-42a6-b2a8-50b204a6d0e3.png">

**Comment by yunwu-cn** [comment]
The "All Tests" summary is correct, when a specific tag is selected, the stats are incorrect.
